### ISSUE TITLE
QB-Target Refuel Menu Issues (Only working on one side)

### ISSUE THREAD

**Issue by dnelyk** [CodineDev/cdn-fuel#3]
Although this feature is kind of cool, **it is not intended**. <br> <br> We are actively working on a fix. The fix will include more steps in the installation 😢 but will be better in the long run!

**Comment by dnelyk** [comment]
Fixed in Version 1.0.2 now with a Config Option & Install Guide!

**Comment by dnelyk** [comment]
### QB-Target Issue Fix 

There is a **possible** issue with *qb-target* if you are using the *Config.GlobalVehicleOptions* or *Config.TargetBones* options. 
<br>
### **If you are NOT having this issue occur, do not follow the instructions below, as it could mess up other things.**
<br>

*Here is a simple fix for that issue:*

<br> 

Firstly, this option will have to be added to your *Config.TargetBones* under the bones you are having trouble with:
```
            {
				type = "client",
				event = "cdn-fuel:client:SendMenuToServer",
				icon = "fas fa-gas-pump",
				label = "Insert Nozzle",
				canInteract = function() return Allowrefuel end
            },
```

*Here is an example of how to add this option:*

![Step5part33 QB-Target](https://i.imgur.com/UOgPJRi.png)
<br> 
*This is **specifically** for the "**boot**" bone, but, add it on which bone you are having trouble with.*

<br>
<br>

*Next, we'll add this simple Function & Export into our QB-Target in the Functions() area:*

```
local function AllowRefuel(state) 
    if state then
        Allowrefuel = true
    else
        Allowrefuel = false
    end
end exports('AllowRefuel', AllowRefuel)
```

<br> 

**Example Image:**

![Step5 Part 421421412](https://i.imgur.com/pwpa5Tk.png)

Now, set the *Config.FuelTargetExport* in *cdn-fuel/shared/config.lua* to **true**.

<br> 

![Step5 Part 1421942151251](https://i.imgur.com/InBl500.png)
